packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -117,6 +117,7 @@ const knownOptions = {
   import: ['string'],
   exhaustive: ['boolean'],
   query: ['object', 'string'],
+  base: ['string'],
```

Comment:
It would be nice if we can throw an error for `base` starting with `!` so that we can tell that prepending `!` with `base` option is not supported.

Reply:
I fixed [ae89d14](https://github.com/vitejs/vite/pull/18510/commits/ae89d14861d35f81a512a27e82b0ea87b53a793a)